MyCryptoHQ/MyCrypto
# Migrate storage from `beta.mycrypto.com`

Requires https://github.com/MyCryptoHQ/landing/pull/12 to be merged in Landing.  
Replaces https://github.com/MyCryptoHQ/MyCrypto/pull/3415

## Steps to test:
[To test locally, use `https://localhost:8000` for landing and `https://localhost:3000` for app.]
1. Head to [`landing.mycryptobuilds.com`](https://landing.mycryptobuilds.com).
2. Open the console and test that `document.domain` is correctly set to `mycryptobuilds.com`.
3. Add a `v1.0.0` LS with the proper key eg. `MYC_Storage`.
4. Open this PR [link](https://mycryptobuilds.com/pr/3430) in a new tab.
5. Open the console and test that `document.domain` is the identical to the landing page.
6. Open `DevMode` and Reset your localStorage. (this is irreversible so make sure you have a backup before)
7. Refresh your page. (You should arrive to `/no-accounts`
8. See that the banner appears (Should we do a Modal instead?)
9. Click "Yes".
10. Expect: 
- To be redirected to `/dashboard` with your accounts
- See in devTools that the LS updated. You may need to wait. It will take a max of 30secs to see the change.
- See that the storage in landing is deleted.

## Missing:
- [ ] Correct UX for user
